modules/flowable-cmmn-converter/src/test/java/org/flowable/test/cmmn/converter/ParentCompletionConverterTest.java

```diff
@@ -57,10 +57,11 @@ public void validateModel(CmmnModel cmmnModel) {
         Stage stageOne = (Stage) cmmnModel.getPrimaryCase().getPlanModel().findPlanItemDefinitionInStageOrDownwards("stageOne");
         List<PlanItem> planItems1 = stageOne.getPlanItems();
         assertThat(planItems1).hasSize(1);
-        PlanItem planItem = planItems1.get(0);
-        assertThat(planItem.getItemControl()).isNotNull();
-        assertThat(planItem.getItemControl().getParentCompletionRule()).isNotNull();
-        assertThat(planItem.getItemControl().getParentCompletionRule().getType()).isEqualTo(ParentCompletionRule.IGNORE_AFTER_FIRST_COMPLETION);
+        assertThat(planItems1)
```

Comment:
The extracting from assertJ is really powerful, but I don't think that ti fits the purpose everywhere. In this case the new assertions are more difficult to follow (at least to me)

Reply:
@filiphr I undid the change; so be good to go.